aostypes/aostypes.go

```diff
@@ -128,6 +128,7 @@ type ServiceInfo struct {
 	VersionInfo
 	ID         string `json:"id"`
 	ProviderID string `json:"providerId"`
+	GID        uint32 `json:"gid"`
```

Comment:
Do we need it here? SM can take it itself from service config.

Reply:
GID is generated by imagemanager.